xrally_kubernetes/common/opts.py

```diff
@@ -0,0 +1,40 @@
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from rally.common import cfg
+
+
+KUBERNETES_OPTS = [
+    cfg.FloatOpt("start_prepoll_delay",
+                 default=0.0,
+                 deprecated_group="benchmark",
+                 help="Time to sleep after start before polling for status"),
```

Comment:
Replace `after start` with `after performing the previous action` or remove it at all.

Reply:
OK